rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -1160,6 +1161,40 @@ def test_upgrade_db_21_to_22(user_data_dir):  # pylint: disable=unused-argument
     assert db.get_version() == 22
 
 
+def test_upgrade_db_22_to_23(user_data_dir):  # pylint: disable=unused-argument
```

Comment:
Test is good! This is for the case the user has the settings populated in their DB. In the case it's missing we probably also need one as I just saw the upgrade code had an error which that test would catch.

Ofcourse upgrades happen only once in the lifetime of the app so the test won't be that useful after next version, but better have it to be on the safe side.